src/engraving/libmscore/score.cpp

```diff
@@ -2764,6 +2764,46 @@ void Score::adjustKeySigs(int sidx, int eidx, KeyList km)
     }
 }
 
+//---------------------------------------------------------
+//   getKeyList
+//      This is taken from MuseScore::editInstrList()
+//---------------------------------------------------------
+
+KeyList Score::getKeyList() const
```

Comment:
Better to rename this function to keyList() (code style is the reason)

Reply:
Done.